src/api/publicationApi.ts

```diff
@@ -57,8 +47,14 @@ export const createNewPublication = async (
 };
 
 export const updatePublication = async (publication: Publication, dispatch: Dispatch) => {
-  const { publicationIdentifier } = publication;
-  const url = `${API_URL}/update-resource/${publicationIdentifier}`;
+  const { id } = publication;
+  if (!id) {
+    // create new if not existing?
+    dispatch(addNotification(i18n.t('feedback:error.update_publication'), 'error'));
+    return;
+  }
+
+  const url = `${API_URL}/update-resource/${id}`;
```

Comment:
jeg har laget en egen konstant for /channel/search. kanskje trekke ut denne også?